README.md

```diff
@@ -1,243 +1,178 @@
-# Microformats (ruby)
+<img src="https://raw.githubusercontent.com/gist/jgarber623/5c4cb8712169f51b571d370084006699/raw/d5ece174e46d98770a4e66f313e81e37ee3a5bf0/microformats-ruby-logo.svg?sanitize=true" align="left" width="100">
```

Comment:
want to get confirmation from @tantek that it is okay to use the microformats logo with a recolor for a different project.  I can't see there being any complaint, but just want to check

Reply:
Good call. The [microformats wiki's copyright page](http://microformats.org/wiki/copyright) has content dedicated to the public domain _after_ 2007-12-29. I believe the logo was added prior to that, so it'd be a good idea to get @tantek's feedback on this.

Easy enough to leave the logo image out if it's an issue.

Reply:
Removed in 10d2fc0. Will open a separate issue for logo conversations.